hedera-node/test-clients/src/main/java/com/hedera/services/bdd/spec/HapiSpecOperation.java

```diff
@@ -223,6 +223,7 @@ public Optional<Throwable> execFor(final HapiSpec spec) {
                 updateStateOf(spec);
             }
         } catch (final Throwable t) {
+            t.printStackTrace();
```

Comment:
This was unintentional, though I don't know if we still have any concerns about log spam.

Reply:
removed.